pkgs/development/libraries/sycl-info/default.nix

```
@@ -0,0 +1,59 @@
+{ stdenv
+, fetchFromGitHub
+, installShellFiles
+, cmake
+, ninja
+, ocl-icd
+, opencl-headers
+, lyra
+, nlohmann_json
+, ronn
+, doctest
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sycl-info";
+  version = "unstable-2019-11-19";
+
+  src = fetchFromGitHub {
+    owner = "codeplaysoftware";
+    repo = "sycl-info";
+    rev = "b47d498ee2d6b77ec21972de5882e8e12efecd6c";
+    sha256 = "0fy0y1rcfb11p3vijd8wym6xkaicav49pv2bv2l18rma929n1m1m";
+  };
+
+  buildInputs = [
+    nlohmann_json
+    ronn
+    opencl-headers
+    ocl-icd
+    doctest
+    lyra
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+  ];
+
+  enableParallelBuilding = true;
+
+  cmakeFlags = [
+    "-DBUILD_TESTING=ON"
+    "-DBUILD_DOCS=ON"
+    "-DBUILD_SHARED_LIBS=ON"
+    "-DLYRA_INCLUDE_DIRS=${lyra}/include"
```

future proof against multi-output change in the future:
```suggestion
    "-DLYRA_INCLUDE_DIRS=${lib.getDev lyra}/include"
```